server/models/Category.model.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+
+const categorySchema = new mongoose.Schema({
+  category: {
+    type: String,
+    trim: true,
+    uppercase: true,
+    required: true,
+  }
```

Comment:
Sorry I forgot to mention that the Gantt Chart allows users to change the colors of TODO items so it may be best to add a "color" property to Category for this (as opposed to keeping an array with unnecessary duplication in ChartSettings las is done currently). You don't necessarily need to create a view for editing categories since, for now, the color would only be used by the Gantt Chart; however, you would need to create an endpoint for editing category so that its color can be changed.

Reply:
I personally think that the Category model should not need to deal with colour. Cause within TODOList module, categories are not given a color. So this colour matter is mainly a concern for the Gantt Chart, not the TODOList. Can we for this milestone keep it the way it is?

Reply:
For this milestone we could. The main problem is that not attaching color to each category means information about the user's categories would just be duplicated in ChartSettings to associate them with a color which is unnecessary. In the future, other functionality may want to use a user's assigned category color as well so having a centralized place for it would be nice.

Reply:
Ok. I guess for this pull request we can perhaps put this on hold. I will look into this in the future

Reply:
In fact you could even color the TODO cards by the color of the category since the cards don't show category (they only show title, start date, end date)

Reply:
Perhaps we should ask the team what they think.